workload/basic_workload.cpp

```diff
@@ -62,16 +77,17 @@ BasicWorkload::BasicWorkload(
     config_(config),
     partition_to_key_lists_(config->GetNumPartitions()),
     client_txn_id_counter_(0) {
-
+  auto num_replicas = config->GetNumReplicas();
+  auto num_partitions = config->GetNumPartitions();
+  auto hot_keys_per_list = params_.GetUInt32(HOT) / (num_replicas * num_partitions);
   for (auto& key_lists : partition_to_key_lists_) {
-    for (uint32_t rep = 0; rep < config->GetNumReplicas(); rep++) {
-      // TODO (ctring): initialize each key list with some hot keys
-      key_lists.emplace_back();
+    for (uint32_t rep = 0; rep < num_replicas; rep++) {
+      key_lists.emplace_back(hot_keys_per_list);
```

Comment:
A note here might help, otherwise you have to go to this header to see the list type, then find the KeyList constructor